chrome_extension/cardboard/popup.js

```diff
@@ -1,10 +1,37 @@
+const IS_PROD = true;
```

Comment:
Add a TODO to improve this? Feels like it should be `false` by default and then the release process for the extension includes setting it to `true` in some automatic way.